munimap/layers.py

```diff
@@ -781,10 +781,11 @@ def check_project_config(new_yaml_content, exclude_file):
         'layers': [],
         'groups': []
     }
-    if isinstance(new_yaml_content, basestring):
+    if isinstance(new_yaml_content, str):
         return ['String only not supported'], False
 
     config_folder = current_app.config.get('LAYERS_CONF_DIR')
+    print(config_folder)
```

Comment:
Should we rather use the app logger instead of print here? Then all log settings would apply for this line

Reply:
It would be nice to get some feedback on this

Reply:
This is a leftover for a debugging session. No need to be here. I'll remove it